packages/stratego-four/src/game/gameSaga.js

```diff
@@ -24,13 +31,23 @@ function* defaultTxOps() {
  *
  * @param {string} player - The address of the player
  */
-function* initializePlayer(player) {
-  yield put(setPlayer(player));
-
+function* initializePlayer({ player }) {
   // get the ops after setting the player
   const ops = yield defaultTxOps();
 
-  yield put(setCurrentGameKey(stratego4.methods.currentGame.cacheCall(ops)));
+  yield put(
```

Comment:
TIL; javascript has a `yield` 🤔 [I read the docs... ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/yield) neat! 👍

Reply:
Yeah it was another attempt at async before async/await. I expect they'll rewrite this library to use that syntax soonish?

Also, interesting fact. `async/wait` is polified on browsers with generators. hence the name "regenerator runtime"